src/ast/nodes/shared/knownGlobals.ts

```diff
@@ -35,33 +35,33 @@ interface GlobalDescription {
 	__proto__: null;
 }
 
-const getTruthyLiteralValue = (): LiteralValueOrUnknown => UnknownTruthyValue;
+const getUnknownValue = (): LiteralValueOrUnknown => UnknownValue;
```

Comment:
Now, we consider that `UnknownTruthyValue` is true in logical evaluation. To ensure some polyfill JS scripts work properly, we need to change `UnknownTruthyValue` to `UnknownValue`. Perhaps we can return `UnknownTruthyValue` if the `treeshake` option is `'smallest'`?

Reply:
That is probably true. But as before the PR, it did not have an effect, it would not affect anyone if we changed that. I am not sure we can easily read the tree shaking option here, your solution is perfect for now IMO.